light/store/db/db.go

```diff
@@ -214,7 +219,7 @@ func (s *dbs) LightBlockBefore(height int64) (*types.LightBlock, error) {
 // left.
 //
 // Safe for concurrent use by multiple goroutines.
-func (s *dbs) Prune(size uint16) error {
+func (s *dbs) Prune(size uint16) (err error) {
```

Comment:
I'm not really a fan of named return values, it's too easy to get them wrong (e.g. by shadowing the variable and returning `nil` instead of what you think you're returning).